### PR TITLE
Google Cloud Hosting

### PR DESCRIPTION
**Pull request by oddaspa**
- Add Dockerfile for pushing to Clarify
- Add notebook with hosting guide
- Add versioning to requirements, update .gitignore

### CODE REVIEW

**Comment by AlexiaArtemis** [comment]
Nice and I can add on top of it mine changes. 👍

File: tutorials/docker/webapp/main.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    
+    # add metadata for enums
+    signal = Signal(name=SIGNAL_ID_LIST[1], type="enum", enumValues={data[1]: word})
+    response_2 = client.save_signals(inputs={SIGNAL_ID_LIST[1]: signal}, created_only=False)
+    print(response_2)
+
+app = Flask(__name__)
+
+@app.route("/")
+def hello_world():
```
Comment (AlexiaArtemis) [inline tutorials/docker/webapp/main.py:49]: Did you check if this is necessary?

Comment (AlexiaArtemis) [reply tutorials/docker/webapp/main.py:49]: Actually we can leave it for now 👍

Comment (oddaspa) [reply tutorials/docker/webapp/main.py:49]: I forgot to test! I am testing now :)

**Pull request merged**